pkg/report/linux.go

```diff
@@ -1225,6 +1225,7 @@ var linuxStackParams = &stackParams{
 		"print_hex_dump",
 		"^klist_",
 		"(trace|lockdep)_(hard|soft)irq",
+		"netlink_ack",
```

Comment:
Perhaps we want to skip it only for KMSAN reports? I see we already skip alloc_skb only for KMSAN.

Reply:
Done.